port: /dev/ttyUSB2
name : PowerControlBoard 
publish:
 PowerControl: #topic name
  type: power_control_board/PowerControl 
subscribe:
 CutterControl:
  type: power_control_board/CutterControl
